app/controllers/users_controller.rb

```diff
@@ -5,6 +5,5 @@ def index
 
   def show
     @user = User.find(params[:id])
-    @posts = Post.order(created_at: :desc).where(user_id: @user.id)
   end
 end
```

Comment:
- [ ] In the users controller index actions, let us use ``@posts = @user.posts.includes(:comments)`` to solve the n+1 query problem instead of the `@user.post`

Reply:
Fixed